pom.xml

```diff
@@ -138,6 +138,7 @@
     <site.installationModule>google-cloud</site.installationModule>
     <bom.version>0.37.1-alpha-SNAPSHOT</bom.version><!-- {x-version-update:google-cloud-pom:current} -->
     <api-client.version>1.23.0</api-client.version>
+    <gax.version>1.19.0</gax.version>
```

Comment:
I see that this was broken since I created the BOM module. Ideally I'd like to avoid having the gax version specified in more than one place. Let's do this for now, knowing that we need to implement https://github.com/googleapis/gax-java/issues/489 at some point.